### PR TITLE
Eh main/feature/cli consumer

### PR DESCRIPTION
**Pull request by eHattori**
# ✨ New Feature

Create a CLI to load all needed for the application

# 🖼 Screenshots
image

# 🗂 Related cards

[[Async] Create a CLI to load all needed to the application](https://app.pipefy.com/open-cards/529931803)

### CODE REVIEW

**Review by looslidias — APPROVED** [review]

File: Gemfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ source "https://rubygems.org"
 # Specify your gem's dependencies in pipefy_message.gemspec
 gemspec
 
-gem "activesupport", "~> 5.0", ">= 5.0.0.1"
+gem "activesupport", "~> 4.2.11.3"
```
Comment (md-hamed) [inline Gemfile:8]: Just wondering, why do we need ActiveSupport, and why are we using an old version of it?

**Review by md-hamed — APPROVED** [review]

File: Gemfile (hunk 1 above)
Comment (eHattori) [reply Gemfile:8]: The version I forgot to back to the last version, We used to create objects from strings in real time with `.constantize.new` [here](https://github.com/pipefy/pipefy_message/blob/a30e74009b9385f04734b4342224a978c45281bb/lib/pipefy_message/consumer.rb#L59)

I changed the version to test one thing in core and forgot to come back the version

Comment (md-hamed) [reply Gemfile:8]: Got it, thanks @eHattori 

But now that we are on Rails 4, does it make sense to use the Rails 5 version of it?

Comment (eHattori) [reply Gemfile]: done!

**Pull request merged**